.github/workflows/build.yml

```diff
@@ -2,10 +2,11 @@ name: Code Analysis & Test
 
 on:
   push:
+    branches:
+      - master
   pull_request:
-  # Once on the first of the month at 06:00 UTC
-  schedule:
-    - cron: 0 6 1 * *
```

Comment:
I think this code is for early detecting if any dependencies breaking the CI. I think we can remove it